THIRD_PARTY_LICENSES.txt

```diff
@@ -8656,217 +8656,6 @@ publicity pertaining to the work without specific, written prior permission.
 Title to copyright in this work will at all times remain with copyright 
 holders.
 
```

Comment:
why s this getting removed?

Reply:
axios is removed, so it's license is also removed from THIRD_PARTY license text file